README.md

```diff
@@ -111,4 +151,4 @@ Contains images of the generated base45 contents(PNG).
 
 ### JWK Content
 
-The key pair to sign and validate the data structures. This is defined as x5c datastructure which contains the complete key pair.
+The key pair to validate the data structure. This is defined as x5c datastructure which contains the public key.
```

Comment:
Suggestion: The public key to validate the data structure. This is defined as an x5c data structure containing the certificate holding this public key.